GNUmakefile

```diff
@@ -214,22 +215,30 @@ proto:
 		protoc -I . -I ../../.. --go_out=plugins=grpc:. $$file; \
 	done
 
-
+.PHONY: vendorfmt
 vendorfmt:
 	@echo "--> Formatting vendor/vendor.json"
 	test -x $(GOPATH)/bin/vendorfmt || go get -u github.com/magiconair/vendorfmt/cmd/vendorfmt
 		vendorfmt
+
+.PHONY: changelogfmt
 changelogfmt:
 	@echo "--> Making [GH-xxxx] references clickable..."
 	@sed -E 's|([^\[])\[GH-([0-9]+)\]|\1[[GH-\2](https://github.com/hashicorp/nomad/issues/\2)]|g' CHANGELOG.md > changelog.tmp && mv changelog.tmp CHANGELOG.md
 
+## We skip the terraform directory as there are templated hcl configurations
+## that do not successfully compile without rendering
+.PHONY: hclfmt
+hclfmt:
+	@echo "--> Formatting HCL"
+	@find . -path ./terraform -prune -o -name '*.hcl' -name '*.nomad' -exec hclfmt -w {} +
```

Comment:
i think this needs to be:
```
find . -path ./terraform -prune -o \( -name '*.nomad' -o -name '*.hcl' \) -exec hclfmt -w {} +
```

Reply:
oh oops (actually just needs the extra -o, not the parens tho)

Reply:
it played with it some. it seemed to before-coffee me that without the parens, the implicit `-a` between `exec` and the second `name` takes precedence over the `-o`

Reply:
>        Please note that -a when specified implicitly (for example by two
       tests appearing without an explicit operator between them) or
       explicitly has higher precedence than -o.  This means that find .
       -name afile -o -name bfile -print will never print afile.

Reply:
huh til